README.md

```diff
@@ -1 +1,110 @@
-Test
\ No newline at end of file
+# Trustly Parliament Challenge
+
+## Problem
+
+This is a challenge by Trustly known as the "Parliament Challenge".
+
+The Swedish parliament has an open API platform for developers to use its resources to create applications. Among its resources are the speeches in the parliament and the data of its members.
+
+The challenge by Trustly is to create an appliction that serves requests for parliament speeches. The expected response should be created by merging the data from the speeches api and members api and should be in a JSON format.
+
+The expected response should at least contain:
+
+- A unique ID, representing the speech itself (anforande_id)
+- Date of speech (dok_datum)
+- The name of the speaker, and only the name (tilltalsnamn)
+- Political affiliation (parti)
+- The official e-mail address (uppgift)
+- Constituency (valkrets)
+- A decent sized image that could be used in a web site listing (bild_url_192)
+- The debate subject (avsnittsrubrik)
+- A link to the speech (HATEOAS style) (protokoll_url_www)
+
+The link to the challenge can be found on https://github.com/trustly/parliament-challenge
+
+## Approach
+
+The challenge is approached as follows:
```

Comment:
Revise the entire description. Don't go into very technical detailed implementations. Just give a very simple description to your approach on what the api is about and how you can use it. Don't include any function names in your code. Try to just be very high level and simple. Example:

The speech api is .... It contains the following fields .. These fields represent ....
The members api is ..  It contains the following fields .. These fields represent ...

Since the resources from the speech api has member related info, we can use that field to get details of the member using the member api .. 

Once we get that, we can merge it like ... (give an example).

Example request / response from the speech api (omit all fields, show only the ones you're interested in) and just show max two responses and do a `...` after the list to indicate there are more results:
>> ... curl ....
```json
{
}
```


Example request / response from the member api:
>> ... curl ....
```json
{
}
```

Then show an example of the merged result.
```json
{ ...}
```

Reply:
revised approach